### DISCUSSION TITLE
Is it possible to add a bottom bar shadow?

### DISCUSSION THREAD

**Discussion by madalinpopa** [qtile/qtile#3456] [Q&A]
Hi Everyone, 

I'm a new Qtile user. 
I'm looking to see if there is any way to add a bottom border shadow for Qtile bar. Is it possible?

Thanks,

**Comment by m-col** [comment]
Hello! Apologies for forgetting about this. If it's just a drop shadow for the bar that you are after, and you're using X11, then you can get this with `picom`. You'll need to add a rule to the picom config file to match the bar's window, and then enable a shadow for it. Check out https://github.com/qtile/qtile/issues/3236 for more info on matching the window. Hope that helps
